src/DataCollector/PHPCRDataCollector.php

```diff
@@ -183,11 +183,13 @@ public function collect(Request $request, Response $response, \Exception $except
         }
 
         $this->data['documents'] = $documents;
+
+        $this->data = $this->cloneVar($this->data);
```

Comment:
this is only available in sf 3, 2.8 does not have this. and are you sure this is a good idea? we clone data onto itself when we collect? if that is a new requirement, maybe simply check if the method exists and otherwise don't clone as it seems to have worked in sf 2?

Reply:
I borrowed it from the [RequestDataCollector](https://github.com/symfony/http-kernel/blob/master/DataCollector/RequestDataCollector.php#L165).

I will remove this commit and modify our data collector template.